### PR TITLE
fix(#8173): "patch" node-fetch in api and sentinel.

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

Installs latest `node-fetch@2` and "patches" api and sentinel images, so pouchdb-fetch uses the updated dependency. 

medic/cht-core#8173

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs

If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8173-pouchdb-upgrade/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8173-pouchdb-upgrade/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8173-pouchdb-upgrade/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -300,6 +300,7 @@ module.exports = function(grunt) {
               `cd ${service}`,
               `npm ci --production`,
               `npm dedupe`,
+              `rm -rf ./node_modules/pouchdb-fetch/node_modules/node-fetch`,
```
Comment (garethbowen) [inline Gruntfile.js:303]: Comment this so we know why it's there.

```suggestion
              `rm -rf ./node_modules/pouchdb-fetch/node_modules/node-fetch`, // remove buggy version of node-fetch #8173
```

File: haproxy/entrypoint.sh [hunk 2]
```diff
@@ -1,8 +1,6 @@
 #!/bin/bash
 
 set -e
-# Make sure service is running
-service rsyslog start
```
Comment (garethbowen) [inline haproxy/entrypoint.sh:5]: TODO: revert

**Review by garethbowen — APPROVED** [review]
Easy! Nice one. Nitpick inline.

I'll keep working on getting the fix in PouchDB...

File: haproxy/entrypoint.sh (hunk 2 above)
Comment (dianabarsan) [reply haproxy/entrypoint.sh]: reverted, sorry. I had this change locally to test everything works together and committed it accidentally.

**Pull request merged**